.github/workflows/test-all.yml

```diff
@@ -117,7 +117,7 @@ jobs:
           # "bundled" with NormalizInterface, as this is very slow (takes 5-10
           # minutes). Instead we want it to use libnormaliz-dev installed by
           # us
-          rm -f */build-normaliz.sh
+          rm -f **/build-normaliz.sh
```

Comment:
Why is this needed?

Reply:
Probably not needed. I do not know why I thought it was neccessary.